index.html

```diff
@@ -0,0 +1,421 @@
+    <link rel="stylesheet" href="assets/css/footer.css" />
+    <link rel="stylesheet" href="assets/css/sidebar.css" />
+    <link rel="stylesheet" href="assets/css/style.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.5.0/css/all.min.css"
+    />
+  </head>
+  <body>
+    <div class="sidebar">
+      <div class="scroll-line"></div>
+      <ul>
+        <li>
+          <a href="#home"
+            ><i class="fas fa-home"></i> <span class="link-text">Home</span></a
+          >
+        </li>
+        <li>
+          <a href="#about"
+            ><i class="fas fa-user"></i>
+            <span class="link-text">About Me</span></a
+          >
+        </li>
+        <li>
+          <a href="#projects"
+            ><i class="fas fa-briefcase"></i>
+            <span class="link-text">Projects</span></a
+          >
+        </li>
+        <li>
+          <a href="#education"
+            ><i class="fas fa-graduation-cap"></i>
+            <span class="link-text">Education</span></a
+          >
+        </li>
+        <li>
+          <a href="#contact"
+            ><i class="fas fa-envelope"></i>
+            <span class="link-text">Contact</span></a
+          >
+        </li>
+      </ul>
+    </div>
+    <header class="header">
+      <nav class="header-nav">
+        <img
+          class="mut-icon"
+          src="assets/images/muticon.png"
+          alt="Mut Icon not found"
+        />
+        <ul>
+          <li>
+            <a href="https://www.facebook.com/muthane.aboalez" target="_blank"
+              ><i class="fab fa-facebook social-icon"></i
+            ></a>
+          </li>
+          <li>
+            <a href="https://www.instagram.com/muthana_abo_alez" target="_blank"
+              ><i class="fab fa-instagram social-icon"></i
+            ></a>
+          </li>
+          <li>
+            <a href="https://github.com/muthana-abo-alez1" target="_blank"
+              ><i class="fab fa-github social-icon"></i
+            ></a>
+          </li>
+          <li>
+            <a
+              href="https://www.linkedin.com/in/muthana-abo-alez"
+              target="_blank"
+              ><i class="fab fa-linkedin social-icon"></i
+            ></a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <section id="home" class="home">
+        <div class="home-content">
+          <h1>Hello !</h1>
+          <h2>I Am Muthana Abo Aleez</h2>
+          <p>
+            A Dedicated Front-End Developer and Quality Assurance Developer,
+            committed to enhancing user experiences through seamless design and
+            meticulous testing
+          </p>
+          <a
+            class="btn"
+            href="https://drive.google.com/file/d/1b26wvk9MrUTZVXtNsVjSvSiyFwR2U5ed/view?usp=drive_link"
+            target="_blank"
+            rel="noopener noreferrer"
+          >
+            <button class="btn-view-cv">View CV</button>
```

Comment:
You can implement this as "Download CV" in a way that once the URL is clicked, it downloads it as PDF.
If you want it to still navigate to the drive URL, please consider to make the access public.